sdk/eventhubs/azure-messaging-eventhubs-track2-perf/src/main/java/com/azure/messaging/eventhubs/perf/EventProcessorJedisOptions.java

```diff
@@ -0,0 +1,97 @@
+package com.azure.messaging.eventhubs.perf;
+
+import com.azure.perf.test.core.PerfStressOptions;
+import com.beust.jcommander.Parameter;
+
+/**
+ * Options for Event Processor Jedis tests.
+ *
+ * @see EventProcessorJedisTest
+ */
+public class EventProcessorJedisOptions extends PerfStressOptions {
+    @Parameter(names = {"-hn", "--hostName"}, description = "Host name of Azure Redis cache.")
```

Comment:
There's a lot of overlap with EventProcessorOptions and this class...
what about EventProcessorClientOptions with the base options.. then something like EventProcessorClientStorageBlobOptions then yours EventProcessorClientRedisOptions?